include/tvm/ir/expr.h

```diff
@@ -374,6 +374,12 @@ inline Bool operator&&(const Bool& a, const Bool& b) {
   return Bool(a.operator bool() && b.operator bool());
 }
 
+inline Bool operator==(const Bool& a, bool b) { return Bool(a.operator bool() == b); }
```

Comment:
Shall we return bool instead of Bool? Thanks!

Reply:
Good point. Done.